### PR TITLE
OSDOCS2378: Added source files of Egress Lockdown feature

### PR DESCRIPTION
**Pull request by sayjadha**
This PR contains the source file for ARO feature - Egress Lockdown (https://issues.redhat.com/browse/OSDOCS-2378)

### CODE REVIEW

**Comment by sayjadha** [comment]
Note for peer reviewer: 
- This content is created as part of Microsoft ARO work done by Red Hat writers.
- Style guide followed:  [Microsoft's contributor's guide](https://docs.microsoft.com/en-us/contribute/).
- Source language used: Markdown (MD reference available here: https://docs.microsoft.com/en-us/contribute/markdown-reference
- Doc guidelines are available here: https://github.com/openshift/openshift-docs/tree/aro-work

File: OSDOCS2378/concepts-egress-lockdown.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  ```
+If the output displays JSON body showing the domains, then egress lockdown is enabled on the ARO cluster. If an empty output is displayed, then egress lockdown is not enabled on that ARO cluster.
+
+Following are the output examples:
+When egress lockdown is enabled on the ARO cluster:
+  ```$ oc get cluster.aro.openshift.io cluster -o go-template='{{ if .spec.gatewayDomains }}{{ "Egress Lockdown Feature Enabled" }}{{ else }}{{ "Egress Lockdown Feature Disabled" }}{{ end }}{{ "\n" }}
+  ```
+When egress lockdown is disabled on the ARO cluster:
+  ```$ oc get cluster.aro.openshift.io cluster -o go-template='{{ if .spec.gatewayDomains }}{{ "Egress Lockdown Feature Enabled" }}{{ else }}{{ "Egress Lockdown Feature Disabled" }}{{ end }}{{ "\n" }}
+  ```
```
Comment (bennerv) [inline OSDOCS2378/concepts-egress-lockdown.md:36]:
```suggestion
To verify if egress lockdown is enabled on a cluster, run the below command. The command will output `Egress Lockdown Feature Enabled` or `Egress Lockdown Feature Disabled` if the feature is enabled or disabled respectively.  
  ```$ oc get cluster.aro.openshift.io cluster -o go-template='{{ if .spec.gatewayDomains }}{{ "Egress Lockdown Feature Enabled" }}{{ else }}{{ "Egress Lockdown Feature Disabled" }}{{ end }}{{ "\n" }}
  ```
```

Comment (sayjadha) [reply OSDOCS2378/concepts-egress-lockdown.md]: Thank you for the review, @bennerv. I incorporated your feedback, and made some minor edits to the language.

File: OSDOCS2378/howto-create-private-cluster-4x.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+1. **Set the following variables.**
+
+   ```console
+   LOCATION=eastus                 # the location of your cluster
+   RESOURCEGROUP="v4-$LOCATION"    # the name of the resource group where you want to create your cluster
+   CLUSTER=aro-cluster             # the name of your cluster
+   ```
+
+1. **Create a resource group**
```
Comment (mjpytlak) [inline OSDOCS2378/howto-create-private-cluster-4x.md:79]: Same as above. There are multiple steps listed as `1`.

**Pull request closed without merging**